op-service/txmgr/rpc.go

```diff
@@ -0,0 +1,54 @@
+package txmgr
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type SimpleTxmgrAPI struct {
+	mgr *SimpleTxManager
+	l   log.Logger
+}
+
+func (a *SimpleTxmgrAPI) GetMinBaseFee(ctx context.Context) *big.Int {
```

Comment:
Nit, you can remove the unused `ctx` argument in all methods in this file, or replace them with `_` if there are other referenced arguments.